### PR TITLE
revamp basic concepts docs

### PR DESCRIPTION
**Pull request by bcoca**
##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

basic concepts

### CODE REVIEW

File: docs/docsite/rst/shared_snippets/basic_concepts.txt [hunk 1]
```diff
@@ -1,34 +1,56 @@
 Control node
 ============
 
-Any machine with Ansible installed. You can run Ansible commands and playbooks by invoking the ``ansible`` or ``ansible-playbook`` command from any control node. You can use any computer that has a Python installation as a control node - laptops, shared desktops, and servers can all run Ansible. However, you cannot use a Windows machine as a control node. You can have multiple control nodes.
+The machine from which you run the Ansible CLI tools (with Ansible installed), the ``ansible`` or ``ansible-playbook`` binaries. You can use any computer that meets the software requirements as a control node - laptops, shared desktops, and servers can all run Ansible. You can have multiple control nodes, but Ansible itself does not coordinate across them, see AAP for a .
```
Comment (Iheanacho-ai) [inline docs/docsite/rst/shared_snippets/basic_concepts.txt]:
```suggestion
You install Ansible on the machine you run the Ansible CLI tools, the ``ansible`` or ``ansible-playbook`` binaries. You can use any computer that meets the software requirements as a control node - laptops, shared desktops, and servers can all run Ansible. You can have multiple control nodes, but Ansible itself does not coordinate across them, see AAP for a.

```

Also I want to ask what does the writer mean by "see AAP for a"

Comment (Iheanacho-ai) [reply docs/docsite/rst/shared_snippets/basic_concepts.txt]: and the first sentence is a little bit hard to understand

File: docs/docsite/rst/shared_snippets/basic_concepts.txt [hunk 2]
```diff
@@ -1,34 +1,56 @@
...
 
 Managed nodes
 =============
 
-The network devices (and/or servers) you manage with Ansible. Managed nodes are also sometimes called "hosts". Ansible is not installed on managed nodes.
+The target devices (servers, network appliances or any computer) you manage with Ansible. Managed nodes are also sometimes called "hosts". Ansible is not installed on managed nodes.
```
Comment (Iheanacho-ai) [inline docs/docsite/rst/shared_snippets/basic_concepts.txt]:
```suggestion
With Ansible, you manage the target devices (servers, network appliances, or any computer). Managed nodes are also sometimes called "hosts". Ansible is not installed on managed nodes.
```

File: docs/docsite/rst/shared_snippets/basic_concepts.txt [hunk 3]
```diff
@@ -1,34 +1,56 @@
...
 
 Inventory
 =========
 
-A list of managed nodes. An inventory file is also sometimes called a "hostfile". Your inventory can specify information like IP address for each managed node. An inventory can also organize managed nodes, creating and nesting groups for easier scaling. To learn more about inventory, see :ref:`the Working with Inventory<intro_inventory>` section.
+A list of managed nodesi provided by one or more 'inventory sources'. Your inventory can specify information like IP address for each managed node,.organize managed nodes into groups and assigning data specific to the group or host by using variables. To learn more about inventory, see :ref:`the Working with Inventory<intro_inventory>` section. Sometimes an inventory source file is also reffered to as a 'hostfile'.
```
Comment (Iheanacho-ai) [inline docs/docsite/rst/shared_snippets/basic_concepts.txt]:
```suggestion
A list of managed nodes provided by one or more 'inventory sources'. Your inventory can specify information like IP address for each managed node. Organize managed nodes into groups and assign data specific to the group or host using variables. See:ref:`the Working with Inventory<intro_inventory>` section to learn more about inventory. Sometimes an inventory source file is also referred to as a 'hostfile'.
```

File: docs/docsite/rst/shared_snippets/basic_concepts.txt [hunk 4]
```diff
@@ -1,34 +1,56 @@
...
 
-Collections
-===========
+Playbooks
+=========
+
+They contain Plays (which are the basic unit of Ansible exeuction), this is both an 'execution concept' and how we describe the files on which ``ansible-playbook`` operates on.
```
Comment (Iheanacho-ai) [inline docs/docsite/rst/shared_snippets/basic_concepts.txt]:
```suggestion
They contain Plays (the basic unit of Ansible execution), this is both an 'execution concept' and how we describe the files on which ``ansible-playbook`` operates.
		Playbooks are written in YAML and are easy to read, write, share and understand. To learn more about playbooks, see :ref:`about_playbooks`.
```

also where it goes "this is both an `execution concept`", what is both an execution concept ?

File: docs/docsite/rst/shared_snippets/basic_concepts.txt [hunk 5]
```diff
@@ -1,34 +1,56 @@
...
+Playbooks are written in YAML and are easy to read, write, share and understand. To learn more about playbooks, see :ref:`about_playbooks`.
+
+
+Plays
+-----
+
+The playbook object that maps manged nodes (hosts) to tasks, all Ansible execution is run in this context. Contains an ordered list of tasks and can be run repeatedly and loop over the mapped hosts over each task.
```
Comment (Iheanacho-ai) [inline docs/docsite/rst/shared_snippets/basic_concepts.txt]:
```suggestion
The playbook object maps managed nodes (hosts) to tasks. All Ansible execution runs in this context. Contains an ordered list of tasks and can be run repeatedly and looped over the mapped hosts over each task.
Plays can include variables as well as tasks.
```

"Contains an", what contains an ?

File: docs/docsite/rst/shared_snippets/basic_concepts.txt [hunk 6]
```diff
@@ -1,34 +1,56 @@
...
+Plays can include variables as well as tasks.
+
+Roles
+.....
+
+A limited distribution of reusable ansible content (tasks, handlers, variables, plugins, templates and files) that can be referenced inside a Play.
```
Comment (Iheanacho-ai) [inline docs/docsite/rst/shared_snippets/basic_concepts.txt]:
```suggestion
A limited distribution of reusable ansible content (tasks, handlers, variables, plugins, templates, and files) can be referenced inside a Play.
```

File: docs/docsite/rst/shared_snippets/basic_concepts.txt [hunk 7]
```diff
@@ -1,34 +1,56 @@
...
 
-Collections are a distribution format for Ansible content that can include playbooks, roles, modules, and plugins. You can install and use collections through `Ansible Galaxy <https://galaxy.ansible.com>`_. To learn more about collections, see :ref:`collections`.
+
+Tasks
+.....
+The units of action in Ansible, must be contained in a Play, directly or via another playbook ojbect contained in a Play (Role, or imported/included task list file).
```
Comment (Iheanacho-ai) [inline docs/docsite/rst/shared_snippets/basic_concepts.txt]:
```suggestion
The units of action in Ansible must be contained in a Play, directly or via another playbook object contained in a Play (Role, or imported/included task list file).
You can execute a single task once with an ad hoc command ``ansible`` or ``ansible-console`` (both create a virtual Play).
```

File: docs/docsite/rst/shared_snippets/basic_concepts.txt [hunk 8]
```diff
@@ -1,34 +1,56 @@
...
-Tasks
-=====
-
-The units of action in Ansible. You can execute a single task once with an ad hoc command.
+Collections
+===========
 
-Playbooks
-=========
+Collections are a distribution format for Ansible content that can include playbooks, roles, modules, and plugins. You can install and use collections through `Ansible Galaxy <https://galaxy.ansible.com>`_. To learn more about collections, see :ref:`collections`. Unlike roles you can use any part of a collection without importing the whole thing.
```
Comment (Iheanacho-ai) [inline docs/docsite/rst/shared_snippets/basic_concepts.txt]:
```suggestion
Collections are a distribution format for Ansible content that can include playbooks, roles, modules, and plugins. Unlike roles, you can use any part of a collection without importing the whole thing. You can install and use collections through `Ansible Galaxy <https://galaxy.ansible.com>`_. To learn more about collections, see :ref:`collections`.	
```

File: docs/docsite/rst/shared_snippets/basic_concepts.txt [hunk 9]
```diff
@@ -1,34 +1,56 @@
...
 
-Ordered lists of tasks, saved so you can run those tasks in that order repeatedly. Playbooks can include variables as well as tasks. Playbooks are written in YAML and are easy to read, write, share and understand. To learn more about playbooks, see :ref:`about_playbooks`.
+AAP
```
Comment (Iheanacho-ai) [inline docs/docsite/rst/shared_snippets/basic_concepts.txt:67]: sandra mentioned how we dont typically mention AAP in the docs, i just wanted to draw attention to this

File: docs/docsite/rst/shared_snippets/basic_concepts.txt [hunk 10]
```diff
@@ -1,34 +1,56 @@
...
+===
+Ansilbe Automation Platform, this is a product that includes enteprise level features and integrates many tools of the Ansible ecosystem: ansible-core, awx, automation hub, etc.
```
Comment (Iheanacho-ai) [inline docs/docsite/rst/shared_snippets/basic_concepts.txt]:
```suggestion
Ansible Automation Platform is a product that includes enterprise-level features and integrates many tools of the Ansible ecosystem: ansible-core, awx, automation hub, etc.
```

File: docs/docsite/rst/shared_snippets/basic_concepts.txt [hunk 11]
```diff
@@ -1,34 +1,69 @@
...
 
-The units of code Ansible executes. Each module has a particular use, from administering users on a specific type of database to managing VLAN interfaces on a specific type of network device. You can invoke a single module with a task, or invoke several different modules in a playbook. Starting in Ansible 2.10, modules are grouped in collections. For an idea of how many collections Ansible includes, take a look at the :ref:`list_of_collections`.
 
-Tasks
-=====
+Plugins
+=======
+Pieces of code that expand Ansible's core capabilities, they can control how you connect to a managed node (connection plugins),
+manipulate data (filter plugins) and even control what is displayed in the console (callback plugins).
+:ref:`working with plugins`
```
Comment (samccann) [inline docs/docsite/rst/shared_snippets/basic_concepts.txt]:
```suggestion
:ref:`working_with_plugins`
```

File: docs/docsite/rst/shared_snippets/basic_concepts.txt [hunk 12]
```diff
@@ -1,34 +1,69 @@
 Control node
 ============
+The machine from which you run the Ansible CLI tools (``ansible-playbook`` , ``ansible``, ``ansible-vault`` and others).
+You can use any computer that meets the software requirements as a control node - laptops, shared desktops, and servers can all run Ansible.
+Multiple control nodes are possible, but Ansible itself does not coordinate across them, see ``AAP`` for such features.
```
Comment (samccann) [inline docs/docsite/rst/shared_snippets/basic_concepts.txt:5]:
```suggestion
Multiple control nodes are possible, but Ansible itself does not coordinate across them.
```

File: docs/docsite/rst/shared_snippets/basic_concepts.txt [hunk 13]
```diff
@@ -1,34 +1,69 @@
...
 
-Any machine with Ansible installed. You can run Ansible commands and playbooks by invoking the ``ansible`` or ``ansible-playbook`` command from any control node. You can use any computer that has a Python installation as a control node - laptops, shared desktops, and servers can all run Ansible. However, you cannot use a Windows machine as a control node. You can have multiple control nodes.
 
 Managed nodes
 =============
+Also referred to as 'hosts', these are The target devices (servers, network appliances or any computer) you aim to manage with Ansible.
```
Comment (samccann) [inline docs/docsite/rst/shared_snippets/basic_concepts.txt]:
```suggestion
Also referred to as 'hosts', these are the target devices (servers, network appliances or any computer) you aim to manage with Ansible.
```

File: docs/docsite/rst/shared_snippets/basic_concepts.txt [hunk 14]
```diff
@@ -1,34 +1,69 @@
...
+It is also used for assigning groups, that both allow for node selection in the Play and bulk variable assignment.
+To learn more about inventory, see :ref:`the Working with Inventory<intro_inventory>` section. Sometimes an inventory source file is also referred to as a 'hostfile'.
 
-A list of managed nodes. An inventory file is also sometimes called a "hostfile". Your inventory can specify information like IP address for each managed node. An inventory can also organize managed nodes, creating and nesting groups for easier scaling. To learn more about inventory, see :ref:`the Working with Inventory<intro_inventory>` section.
 
-Collections
-===========
+Playbooks
+=========
+They contain Plays (which are the basic unit of Ansible execution), this is both an 'execution concept' and how we describe the files on which ``ansible-playbook`` operates on.
```
Comment (samccann) [inline docs/docsite/rst/shared_snippets/basic_concepts.txt]:
```suggestion
They contain Plays (which are the basic unit of Ansible execution). this is both an 'execution concept' and how we describe the files on which ``ansible-playbook`` operates on.
```

File: docs/docsite/rst/shared_snippets/basic_concepts.txt [hunk 15]
```diff
@@ -1,34 +1,69 @@
...
+
+Handlers
+........
+Special form of a Task, that only execute when notified by a previous task which resulted in a 'changed' status.
 
-Collections are a distribution format for Ansible content that can include playbooks, roles, modules, and plugins. You can install and use collections through `Ansible Galaxy <https://galaxy.ansible.com>`_. To learn more about collections, see :ref:`collections`.
 
 Modules
 =======
+The code or binaries that Ansible copies and executes on each managed node (when needed) to accoplish the action defined in each Task.
```
Comment (samccann) [inline docs/docsite/rst/shared_snippets/basic_concepts.txt]:
```suggestion
The code or binaries that Ansible copies and executes on each managed node (when needed) to accomplish the action defined in each Task.
```

File: docs/docsite/rst/shared_snippets/basic_concepts.txt [hunk 16]
```diff
@@ -1,34 +1,69 @@
...
+Each module has a particular use, from administering users on a specific type of database to managing VLAN interfaces on a specific type of network device.
+You can invoke a single module with a task, or invoke several different modules in a playbook.
+Starting in Ansible 2.10, modules are grouped in collections. For an idea of how many collections Ansible includes, take a look at the :ref:`list_of_collections`.
```
Comment (samccann) [inline docs/docsite/rst/shared_snippets/basic_concepts.txt]:
```suggestion
Ansible modules are grouped in collections. For an idea of how many collections Ansible includes, see the :ref:`list_of_collections`.
```

File: docs/docsite/rst/shared_snippets/basic_concepts.txt [hunk 17]
```diff
@@ -1,34 +1,69 @@
...
 
-The units of code Ansible executes. Each module has a particular use, from administering users on a specific type of database to managing VLAN interfaces on a specific type of network device. You can invoke a single module with a task, or invoke several different modules in a playbook. Starting in Ansible 2.10, modules are grouped in collections. For an idea of how many collections Ansible includes, take a look at the :ref:`list_of_collections`.
 
-Tasks
-=====
+Plugins
+=======
+Pieces of code that expand Ansible's core capabilities, they can control how you connect to a managed node (connection plugins),
+manipulate data (filter plugins) and even control what is displayed in the console (callback plugins).
+:ref:`working_with_plugins`
```
Comment (samccann) [inline docs/docsite/rst/shared_snippets/basic_concepts.txt]:
```suggestion
See :ref:`working_with_plugins` for details.
```

File: docs/docsite/rst/shared_snippets/basic_concepts.txt [hunk 18]
```diff
@@ -1,34 +1,69 @@
...
 
-The units of action in Ansible. You can execute a single task once with an ad hoc command.
 
-Playbooks
-=========
+Collections
+===========
+Collections are a distribution format for Ansible content that can include playbooks, roles, modules, and plugins. You can install and use collections through `Ansible Galaxy <https://galaxy.ansible.com>`_. To learn more about collections, see :ref:`collections`. Colleciton resources can be used independantly and discretely from each other.
```
Comment (samccann) [inline docs/docsite/rst/shared_snippets/basic_concepts.txt]:
```suggestion
Collections are a distribution format for Ansible content that can include playbooks, roles, modules, and plugins. You can install and use collections through `Ansible Galaxy <https://galaxy.ansible.com>`_. To learn more about collections, see :ref:`collections`. Collection resources can be used independently and discretely from each other.
```

File: docs/docsite/rst/shared_snippets/basic_concepts.txt [hunk 19]
```diff
@@ -1,34 +1,69 @@
...
+
 
-Ordered lists of tasks, saved so you can run those tasks in that order repeatedly. Playbooks can include variables as well as tasks. Playbooks are written in YAML and are easy to read, write, share and understand. To learn more about playbooks, see :ref:`about_playbooks`.
+AAP
+===
+Short for 'Ansible Automation Platform'. This is a product that includes enterprise level features and integrates many tools of the Ansible ecosystem: ansible-core, awx, automation hub, etc.
```
Comment (samccann) [inline docs/docsite/rst/shared_snippets/basic_concepts.txt]:
```suggestion
Short for 'Ansible Automation Platform'. This is a product that includes enterprise level features and integrates many tools of the Ansible ecosystem: ansible-core, awx, galaxyNG, and so on.
```

**Review by samccann — APPROVED** [review]

**Pull request merged**